crates/viewer/re_viewer_context/src/view/visualizer_system.rs

```diff
@@ -119,6 +118,10 @@ pub trait VisualizerSystem: Send + Sync + 'static {
         None
     }
 
+    /// Called just as this visualizer is registered.
```

Comment:
```suggestion
    /// Called just as this visualizer is registered.
    ///
    /// This can be useful to register additional component fallbacks that are specific to this visualizer's archetype.
```

or something like that